modules/configure-slack.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-slack.adoc
+:_module-type: PROCEDURE
+[id="configure-slack_{context}"]
+= Configuring Slack
+
+Start by creating a new Slack App, and get the webhook URL.
+
+.Procedure
+. Create a new Slack App:
++
+[NOTE]
+====
+If you want to use an existing Slack App, go to https://api.slack.com/apps and select an app.
+====
+.. Navigate to https://api.slack.com/apps/new.
+.. Enter the *App Name* and choose a *Development Slack Workspace* to install your app.
+.. Click *Create App*.
+. On the settings page, *Basic Information* section, select *Incoming Webhooks* (under *Add features and functionality*).
+. Turn on the *Activate Incoming Webhooks* toggle.
+. Select *Add New Webhook to Workspace*.
+. Choose a *channel* that the app will post to, and then select *Authorize*.
+The page refreshes and you are sent back to your app settings page.
+. *Copy* the *Webhook URL*, under the *Webhook URLs for Your Workspace* section.
```

Comment:
We don't markup commands like "Copy" and here, I don't think you could generalize the first "webhook URL" because I don't think it's a title or label in the GUI, though maybe I'm wrong.

```suggestion
. Copy the webhook URL located in the *Webhook URLs for Your Workspace* section.
```